src/OpenGL/Extensions/Silk.NET.OpenGL.Extensions.ImGui/PublicAPI/net5.0/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,18 @@
 #nullable enable
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiController
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.Dispose() -> void
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.Render() -> void
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.Update(float deltaSeconds) -> void
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig.FontSize.get -> int
+Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig.ImGuiFontConfig() -> void
+Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate
+Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate.R = 32882 -> Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate
+Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate.S = 10242 -> Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate
+Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate.T = 10243 -> Silk.NET.OpenGL.Extensions.ImGui.TextureCoordinate
+~Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.ImGuiController(Silk.NET.OpenGL.GL gl, Silk.NET.Windowing.IView view, Silk.NET.Input.IInputContext input) -> void
+~Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.ImGuiController(Silk.NET.OpenGL.GL gl, Silk.NET.Windowing.IView view, Silk.NET.Input.IInputContext input, Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig imGuiFontConfig) -> void
+~Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.ImGuiController(Silk.NET.OpenGL.GL gl, Silk.NET.Windowing.IView view, Silk.NET.Input.IInputContext input, Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig? imGuiFontConfig = null, System.Action onConfigureIO = null) -> void
+~Silk.NET.OpenGL.Extensions.ImGui.ImGuiController.ImGuiController(Silk.NET.OpenGL.GL gl, Silk.NET.Windowing.IView view, Silk.NET.Input.IInputContext input, System.Action onConfigureIO) -> void
+~Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig.FontPath.get -> string
+~Silk.NET.OpenGL.Extensions.ImGui.ImGuiFontConfig.ImGuiFontConfig(string fontPath, int fontSize) -> void
```

Comment:
This doesn't seem right... Can you move this back to the Shipped file? (in all OpenGL files, OpenGLES should be unshipped)

Reply:
i just ran the command again, i guess it did that automatically though? ill fix it rq